app/views/recipe/index.html.erb

```diff
@@ -0,0 +1,31 @@
+    <%= stylesheet_link_tag "food", "data-turbo-track": "reload" %>
+        <%= stylesheet_link_tag "recipe", "data-turbo-track": "reload" %>
+<h1>My Recipes</h1>
+
+<div class='no-recipes'>
+  <%= "No recipes available." if @recipes.empty? %>
+</div>
+
+<div class='add-recipe'>
+  <%= link_to "Add Recipe", new_recipe_path, class: "btn-add-recipe btn-addfood" %>
+</div>
+
+<% if @recipes.any? %>
+  <ul class="recipe-list">
+    <% @recipes.each do |recipe| %>
+      <li>
+        <div class="recipe-info">
+          <div class="recipe-name">
+            <%= link_to recipe.name, recipe_path(recipe), class: "recipe-name" %>
+          </div>
+          <div class="recipe-actions">
+            <%= link_to "Remove", recipe_path(recipe), method: :delete, data: { turbo_method: :delete, turbo_confirm: "Are you sure you want to delete this recipe?" }, class: "btn-remove-recipe" %>
+          </div>
```

Comment:
[REQUIRED] Kindly ensure that only the creator of a recipe has the ability to delete a recipe. This is a requirement for this milestone